internal/sidecar/service/reclaimspace.go

```diff
@@ -149,9 +153,17 @@ func (rs *ReclaimSpaceServer) NodeReclaimSpace(
 			AccessMode: &csi.VolumeCapability_AccessMode{
 				Mode: csiMode,
 			},
+			AccessType: &accessType,
```

Comment:
common type for AccessType is not export, that's why its changed below when volumeMode is block.